PL/VR-PL-0000/rule.json

```diff
@@ -2,26 +2,26 @@
   "Identifier": "VR-PL-0000",
   "Type": "Acceptance",
   "Country": "PL",
-  "Version": "1.0.0",
+  "Version": "1.1.0",
   "SchemaVersion": "1.0.0",
   "Engine": "CERTLOGIC",
   "EngineVersion": "0.7.5",
   "CertificateType": "Vaccination",
   "Description": [
 	{
       "lang": "pl",
-      "desc": "Akceptowane są tylko szczepionki z zestawu wartości, które zostały zatwierdzone przez EMA: Comirnaty, Moderna (Spikevax), Vaxzevria (AstraZeneca), Jannsen."
+      "desc": "Akceptowane są tylko szczepionki z zestawu wartości, które zostały zatwierdzone przez EMA: Comirnaty, Moderna (Spikevax), Vaxzevria (AstraZeneca), Jannsen, Nuvaxovid."
     },    
 	{
       "lang": "en",
       "desc": "Only vaccines in the allowed valueset that have been approved by the EMA are allowed."
     },
     {
       "lang": "de",
-      "desc": "Nur die folgenden Impfstoffe werden akzeptiert: AstraZeneca, Biontech, Janssen, Moderna."
+      "desc": "Nur die folgenden Impfstoffe werden akzeptiert: Vaxzevria (AstraZeneca), Comirnaty, Janssen, Moderna (Spikevax), Nuvaxovid."
```

Comment:
You could consider putting "(Pfizer)" behind "Comirnaty" for consistency.

Reply:
done